ltclient/src/pages/Login.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { Link, browserHistory } from 'react-router';
+import Loading from '../components/Loading';
+import { login, fetchUserVotedTalks } from '../actions';
+
+class Login extends Component {
+
+  state = {
```

Comment:
keep using Prop validation is a good code practice. why ignore for this component ?